src/content/docs/zh-cn/reference/integrations-reference.mdx

```diff
@@ -272,6 +294,18 @@ export default {
 
 ```
 
+### `astro:build:generated`
+
+**上一个钩子**： [`astro:build:setup`](#astrobuildsetup)
+
+**何时**：在静态生产构建完成生成路由和资源之后。
+
+**用途**：在清理构建产物**之前**访问生成的路由和资源。这是一个非常不常见的用例。我们建议使用[`astro:build:done`](#astrobuilddone)，除非你真的需要在清理前访问生成的文件。
+
+```js
+'astro:build:generated'?: (options: { dir: URL }) => void | Promise<void>;
+```
+
 ### `astro:build:ssr`
 
 **上一个钩子**：[`astro:build:setup`](#astrobuildsetup)
```

Comment:
Two lines below this also needs updating, I think, see: https://github.com/withastro/docs/commit/5197abc4c55199e622cc94f6f11e5a1847937873#diff-1fee61b9ccdb2e3f9f0d612dd631d73f5d107244c69d9bf2f446a29702f29de7R293